TGU

TheSwain opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

TheSwain commented:
Hey there, thanks for the PR - was wondering about the stat < UNCONSCIOUS addition. I intended for SetUnconscious() to happen even if you're asleep so that it can update your sleep to 10 seconds if it was below that. As in, if you were a second from awakening, then SetUnconscious() should reset you to 10 seconds (but it doesn't affect you if you're sleep is set to longer than 10 seconds).

Just looking for your intention with the fix. Thanks!

checkraisefold commented:
well, apparently when I was dead the sleeping logo would magically fucking pop up

TheSwain commented:
Nope, you're wrong. That or you don't know the difference between "greater than", "less than", and GEqual/LEqual.

"_target.stat < UNCONSCIOUS_" means "if target.stat is less than 2".  I need it to include UNCONSCIOUS (2), as well as Soft Crit (1) and Conscious (0). If I use < then it does not include Unconscious. 

I think what you're trying to say is that it should be set to  "_target.stat <= UNCONSCIOUS_", which means "less than or equal to UNCONSCIOUS".  This way, this condition excludes is DEAD(3) but lets the other three stat conditions through.

You are completely right about the ZZZ icon though, I just tested it and being dead doesn't prevent the icon from appearing. So I'll stick a condition in there. Thanks for spotting that bug for me.

AnturK committed:
Fixes mood events keeping hard references in category

vuonojenmustaturska committed:
wtf

zeroisthebiggay committed:
huds (#43882)

tgstation-server committed:
Automatic changelog generation for PR #43882 [ci skip]

vuonojenmustaturska committed:
Fixes harddel causes for various things (#43989)

* hardddel stuff

* mech stuff

vuonojenmustaturska committed:
Remove nuclear operative TC consoles in favor of them just getting the TC pre-distributed (#43930)

About The Pull Request

With this PR, the nukies get their TC pre-distributed instead of having to do a complicated song-and-dance routine where people who sort-of know what they're trying try to convince people who don't to put their uplinks into the machines at roughly the same time to press some buttons.
Why It's Good For The Game

The TC donation/distribution cycle is completely unneeded, realistically everyone is going to distribute their TC.

tgstation-server committed:
Automatic changelog generation for PR #43930 [ci skip]

optimumtact committed:
Merge pull request #43987 from AnturK/mood_runtime

Fixes mood events keeping hard references in category

tgstation-server committed:
Automatic changelog compile, [ci skip]

imsxz committed:
cakehat nerf

Partheo committed:
Updates scrubbers (#43856)

tgstation-server committed:
Automatic changelog generation for PR #43856 [ci skip]

ExcessiveUseOfCobblestone committed:
Update mob_helpers.dm

CrazyClown12 committed:
changes sleepy pens to be slightly less bad, same with emagged borg cookies, removes chloralhydratedelayed (#43973)

* changes sleepy pens to be slightly less bad

They have special snowflake chloral hydrate that is literally chloral hydrate but worse. Let's make it chloral hydrate but literally chloral hydrate. Maybe then people will actually use the chems in a sleepy pen instead of emptying it and bitching about how bad it is on discord.

* nerfs the cakehat (not a shitpost)

It's pretty retarded that a single candle on a meme hat is treated as an infinite fuel welder

* Revert "Merge pull request #1 from CrazyClown12/CrazyClown12/cakehat"

This reverts commit 4806c57780a3d6f564b2ec58685b3c9005642892, reversing
changes made to 95b82ed3fd48f7ac7e7ce06fd76afb03860b3e8d.

* remove chloralhydratedelayed

I'm gonna fuck this up again

* delete chloral hydrate delayed from reagents

tgstation-server committed:
Automatic changelog generation for PR #43973 [ci skip]

Commit:
Hangar Bay Shutters need only general CentCom access to open

optimumtact committed:
Merge pull request #44006 from imsxz/hatecak

cakehat nerf

tgstation-server committed:
Automatic changelog generation for PR #44006 [ci skip]

vuonojenmustaturska committed:
Update cryo.dm

vuonojenmustaturska committed:
uplinks for reinforcements

vuonojenmustaturska committed:
real names too

vuonojenmustaturska committed:
Removes commented-out what-if atmos code (#43988)

* meme removal

* Update reactions.dm

zxaber committed:
Reworks AI backup battery system slightly, and allows (un)bolting while on backup at heavy power cost (#43952)

* power underwhelming

* Update ai_core.dm

* status page update

backup power now only shows in the status page if the AI is in the core.

tgstation-server committed:
Automatic changelog generation for PR #43952 [ci skip]

vuonojenmustaturska committed:
goastbrains

ninjanomnom committed:
Adds a source var to a signal receiver that should have it

ninjanomnom committed:
Adds ninjanomnom as an owner of component code for notifications

erwgd committed:
Adds potassium iodide to NanoMed Plus (#43963)

* Added antihol bottle, corazone pill, potass_iodide pill and pill bottle
Added Potassium iodide bottle and pills  to smartfridge preload
Added medicines to NanoMed and NanoMed Plus

* Added small number of potass iodide pills to NanoMed Plus
Removed other medicines being added to vendors in last commit

* Added potass iodide bottle to NanoMed Plus, removed epinephrine bottle and potass iodide pills

* removed unused prebuilt meds

tgstation-server committed:
Automatic changelog generation for PR #43963 [ci skip]

4dplanner committed:
Sets firer for glass golem (#44010)

AnturK committed:
can i have -Wall for dm already (#44008)

vuonojenmustaturska committed:
Fixes colored items like scarves and beanies being shown as white in vending machines (#44007)

* scorf

* whoops

tgstation-server committed:
Automatic changelog generation for PR #44007 [ci skip]

vuonojenmustaturska committed:
Fixes wabbajack altar becoming too inert (#44002)

RaveRadbury committed:
[READY] pAI HUD Overhaul (#43741)

* Initial overhaul work and some fixes.

Running into a bug with getting healthanalyzer variables to pass.

Signed-off-by: RaveRadbury <risforradbury@gmail.com>

* Fixes some typos, shortens a message

* Update pai.dm

* Prevents pAI OS from coming up in other windows.

* Moved around some camera stuff, trying to fix lost pAI zoom

* Fixes pAI zoom, adds camera flash

* PAI buttons grey out if they need software first

* Better greying out of buttons in PAI

* Removes unimplemented HUD stuff

* Should make pAI Host Scan require being held (like DNA check)

* Includes host scan as a greyed out hud

* Improved button loop

* Fixes pda icon off-set

* Fixed inconsistent indentation

* Replaces outdated and innacurate checks for scans

Thanks to Cobblestone for the help

* Refactor pAI hud button grey out

* Changes browse to browser

* cleans up the code, gets rid of an error

* Removes indent from #undef

* Removes an unneeded text-align:right

tgstation-server committed:
Automatic changelog generation for PR #43741 [ci skip]

tgstation-server committed:
Automatic changelog compile, [ci skip]

optimumtact committed:
Merge pull request #44011 from ExcessiveUseOfCobblestone/starsnstripes

[S] The Pen is Mighter Than The Server No Longer

Cyberboss committed:
Merge pull request #44025 from vuonojenmustaturska/reinforcements

0 TC uplinks and real names for reinforcements

tgstation-server committed:
Automatic changelog generation for PR #44025 [ci skip]

Cyberboss committed:
Merge pull request #44035 from ninjanomnom/component-owners

Changes component code owners so I get notified as well

Cyberboss committed:
Merge pull request #44034 from ninjanomnom/squeaky-source

Adds a source var to a signal receiver that should have it

Cyberboss committed:
Merge pull request #44016 from vuonojenmustaturska/cryoicon

Updates cryo tube appearance when a mob exits via teleportation

tgstation-server committed:
Automatic changelog generation for PR #44016 [ci skip]

vuonojenmustaturska committed:
Fixes a bunch of harddels and weirdness. (#44012)

* grabbag of random things

* followup

* deleet

tgstation-server committed:
Automatic changelog generation for PR #44012 [ci skip]

MrFluffster committed:
Adds foodtype CLOTH, gives it to dummy item for clothing eating and makes moths like CLOTH (#43965)

tgstation-server committed:
Automatic changelog generation for PR #43965 [ci skip]

Cyberboss committed:
Merge pull request #43991 from vuonojenmustaturska/vorecode

Removes vorecode

tgstation-server committed:
Automatic changelog generation for PR #43991 [ci skip]

Cyberboss committed:
Merge pull request #44013 from loserXD/mm

Hangar Bay Shutters need only general CentCom access to open

tgstation-server committed:
Automatic changelog generation for PR #44013 [ci skip]

nemvar committed:
Makes podcloning transfer quirks. (#44001)

* Lolwut

* Update other_reagents.dm

tgstation-server committed:
Automatic changelog generation for PR #44001 [ci skip]

AnturK committed:
Machine deconstruction fix

optimumtact committed:
Merge pull request #44028 from vuonojenmustaturska/braindamaged-corpses

Stops corpses from receiving new information about their damaged brain

tgstation-server committed:
Automatic changelog generation for PR #44028 [ci skip]

vuonojenmustaturska committed:
removes an explicit uplink component check in favor of a signal in detomatix carts (#44023)

* yeets a GET_COMPONENT_FROM

* signal rather than component

* Update code/game/objects/items/devices/PDA/virus_cart.dm

Co-Authored-By: Emmett Gaines <ninjanomnom@gmail.com>

tgstation-server committed:
Automatic changelog generation for PR #44023 [ci skip]

Incoming5643 committed:
Syndicate bomb bugfixes and Quality of Death changes (#43996)

tgstation-server committed:
Automatic changelog generation for PR #43996 [ci skip]

kriskog committed:
Fixes some missing adjancency checks. (#44020)

tgstation-server committed:
Automatic changelog generation for PR #44020 [ci skip]

actioninja committed:
Gamemode Announcement Code Cleanup (#43993)

* cleanup

* readd announce

optimumtact committed:
Merge pull request #44051 from AnturK/machinefix

Machine deconstruction fix
